examples/run_example.py

```diff
@@ -85,11 +85,21 @@
         wave_bundle = d.get_waveforms(useful_event)
 
         # print the average snr across channels 
-        print(snr.get_avg_snr(wave_bundle))      
+        avg_snr = snr.get_avg_snr(wave_bundle, excluded_channels=d.excluded_channels)
+        print(f"The Average SNR is {avg_snr:.1f}")
  
         # run our standard suite of reconstructions
         reco_results = reco.do_standard_reco(wave_bundle)
 
+        # here's how we can lookup arrival times given a reconstructed direction
+        arrival_time = reco.lookup_arrival_time(channel = 0, 
+                                                theta = reco_results["pulser_v"]["theta"], 
+                                                phi = reco_results["pulser_v"]["phi"],
+                                                which_distance="distant",
```

Comment:
I feel like this is a silly question on my side but can you remind me why we're paying pulser reconstructions results with `which_distance="distant"`?

Reply:
Because.... I made a mistake. Fixed now haha! Thank you for catching this!